lms/djangoapps/courseware/features/homepage.feature

```diff
@@ -1,12 +1,12 @@
 @shard_1
 Feature: LMS.Homepage for web users
   In order to get an idea what edX is about
-  As a an anonymous web user
+  As an anonymous web user
   I want to check the information on the home page
 
-  Scenario: User can see the "Login" button
+  Scenario: User can see the "Sign in" button
```

Comment:
Is Griff aware of the "Sign in"/"Log out" inconsistency? I'm not sure how that issue was resolved, if at all.